dizoo/box2d/car_racing/entry/my.py

```diff
@@ -0,0 +1,7 @@
+import gym
```

Comment:
remove your own test code in committed codes

Reply:
Deleted.